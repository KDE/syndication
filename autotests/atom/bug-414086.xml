<?xml version="1.0" encoding="utf-8"?>

<feed xmlns="http://www.w3.org/2005/Atom">

<title type="html">Neues von nasauber.de</title>
<subtitle type="html">Tobias Leupolds Meinung zum Weltgeschehen ;-)</subtitle>
<link href="http://nasauber.de/" />
<link rel="self" type="application/atom+xml" href="http://nasauber.de/blog/weblog.xml" />
<icon>http://nasauber.de/bilder/favsonne.png</icon>
<updated>2019-08-25T20:44:44+01:00</updated>
<author><name>Tobias Leupold</name></author>
<rights>Copyright (c) 2019 by Tobias Leupold</rights>
<id>http://nasauber.de/blog/weblog.xml</id>

<entry>
<title type="html">Calculating the difference between two QDates</title>
<published>2019-08-25T20:42:41+01:00</published>
<updated>2019-08-25T20:44:44+01:00</updated>
<id>http://nasauber.de/blog/text.php?text=168</id>
<link href="http://nasauber.de/blog/text.php?text=168" />
<content type="html"> I wanted to calculate the difference between two QDates. Not only the days, but also the years, months, and weeks (for use in KPhotoAlbum). I ended up using the following algorithm: struct &amp;hellip;</content>
</entry>

</feed>
