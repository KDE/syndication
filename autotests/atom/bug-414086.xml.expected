# Feed begin ######################
title: #Neues von nasauber.de#
link: #http://nasauber.de/#
description: #Tobias Leupolds Meinung zum Weltgeschehen ;-)#
copyright: #Copyright (c) 2019 by Tobias Leupold#
# Person begin ####################
name: #Tobias Leupold#
# Person end ######################
# Image begin #####################
url: #http://nasauber.de/bilder/favsonne.png#
# Image end #######################
# Item begin ######################
id: #http://nasauber.de/blog/text.php?text=168#
title: #Calculating the difference between two QDates#
link: #http://nasauber.de/blog/text.php?text=168#
content: #I wanted to calculate the difference between two QDates. Not only the days, but also the years, months, and weeks (for use in KPhotoAlbum). I ended up using the following algorithm: struct &hellip;#
datePublished: #Sun Aug 25 19:42:41 2019#
dateUpdated: #Sun Aug 25 19:44:44 2019#
# Person begin ####################
name: #Tobias Leupold#
# Person end ######################
# Item end ########################
# Feed end ########################